src/styles/blocks/main.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  --column-count: 1;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), custom properties (CSS variables) should be defined in `:root` or in the file with variables. It's better to move `--column-count` to `src/styles/utils/variables.scss` to improve maintainability and reusability of the variables.